### ISSUE TITLE
image for openapimock seems to be broken

### ISSUE THREAD

**Issue by derberg** [kyma-incubator/varkes#123]
Hey,

I tried to follow https://github.com/kyma-incubator/varkes/blob/master/openapi-mock/README.md and ran `docker run -p 10000:10000 eu.gcr.io/kyma-project/incubator/develop/varkes-openapi-mock:latest` but I get error 

```
internal/modules/cjs/loader.js:611
    throw err;
    ^

Error: Cannot find module '/app/start'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:609:15)
    at Function.Module._load (internal/modules/cjs/loader.js:535:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:797:12)
    at executeUserCode (internal/bootstrap/node.js:526:15)
    at startMainThreadExecution (internal/bootstrap/node.js:439:3)
```

**Comment by atakanyenel** [comment]
there seems to be an error with the dockerfile, PR #103 will fix it. Thanks for the effort

**Comment by atakanyenel** [comment]
fixed with #103
